[2025-09-14T14:05:55.959+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-14T14:05:56.142+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_to_postgres_batch.wait_for_kafka scheduled__2025-09-14T13:35:42.545199+00:00 [queued]>
[2025-09-14T14:05:56.157+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_to_postgres_batch.wait_for_kafka scheduled__2025-09-14T13:35:42.545199+00:00 [queued]>
[2025-09-14T14:05:56.158+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 3
[2025-09-14T14:05:56.187+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonSensor): wait_for_kafka> on 2025-09-14 13:35:42.545199+00:00
[2025-09-14T14:05:56.213+0000] {standard_task_runner.py:57} INFO - Started process 12386 to run task
[2025-09-14T14:05:56.226+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_to_postgres_batch', 'wait_for_kafka', 'scheduled__2025-09-14T13:35:42.545199+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpk6468ggb']
[2025-09-14T14:05:56.230+0000] {standard_task_runner.py:85} INFO - Job 150: Subtask wait_for_kafka
[2025-09-14T14:05:56.269+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-14T14:05:56.346+0000] {task_command.py:410} INFO - Running <TaskInstance: kafka_to_postgres_batch.wait_for_kafka scheduled__2025-09-14T13:35:42.545199+00:00 [running]> on host 3deaa5b4a23c
[2025-09-14T14:05:56.376+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-14T14:05:56.574+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kafka_to_postgres_batch' AIRFLOW_CTX_TASK_ID='wait_for_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-09-14T13:35:42.545199+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-14T13:35:42.545199+00:00'
[2025-09-14T14:05:56.576+0000] {python.py:72} INFO - Poking callable: <function check_kafka_message at 0x7fab4b25ff80>
[2025-09-14T14:05:56.581+0000] {conn.py:397} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.23.0.4', 29092)]>: connecting to kafka:29092 [('172.23.0.4', 29092) IPv4]
[2025-09-14T14:05:56.893+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:29092 <checking_api_versions_recv> [IPv4 ('172.23.0.4', 29092)]>: Broker version identified as 2.6
[2025-09-14T14:05:56.894+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.23.0.4', 29092)]>: Connection complete.
[2025-09-14T14:05:56.897+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('user_events',)
[2025-09-14T14:05:56.986+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('user_events', 0)]
[2025-09-14T14:05:56.988+0000] {conn.py:397} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.23.0.4', 29092)]>: connecting to kafka:29092 [('172.23.0.4', 29092) IPv4]
[2025-09-14T14:05:56.989+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:29092 <connected> [IPv4 ('172.23.0.4', 29092)]>: Connection complete.
[2025-09-14T14:05:56.990+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.23.0.4', 29092)]>: Closing connection. 
[2025-09-14T14:05:57.036+0000] {fetcher.py:418} INFO - Resetting offset for partition ('user_events', 0) to offset 0.
[2025-09-14T14:05:57.154+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:29092 <connected> [IPv4 ('172.23.0.4', 29092)]>: Closing connection. 
[2025-09-14T14:05:57.155+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2025-09-14T14:05:57.181+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=kafka_to_postgres_batch, task_id=wait_for_kafka, execution_date=20250914T133542, start_date=20250914T140556, end_date=20250914T140557
[2025-09-14T14:05:57.183+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-09-14T14:05:57.725+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-09-14T14:05:57.816+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
